functions/package-lock.json

```diff
@@ -0,0 +1,3272 @@
+{
```

Comment:
We're using yarn workspaces, so we should use yarn everywhere not `npm`. There should be no `package-lock.json` files or `yarn.lock` files in any of the subdirectories, only `yarn.lock` in the root of the repo.

Reply:
There’s a bit of a challenge getting this one to work that way. I originally used yarn for it, but it’s got to have an “engines” section in the package json witch causes issues with the larger project.